uxarray/helpers.py

```diff
@@ -100,33 +100,6 @@ def parse_grid_type(filepath, **kw):
     return mesh_filetype
 
 
-@njit
-def _spherical_to_cartesian_unit_(node, r=6371):
-    """Converts spherical (lat/lon) coordinates to cartesian (x,y,z).
-
-    Final output is cartesian coordinates on a sphere of unit radius
```

Comment:
Are these cartesian coordinates on a cube that the sphere is enclosed in?

Reply:
we removed the above, these are not on a cube; our intent is just to convert from degree or radians to cartesian for area calculation.